src/meta/ActiveHostsMan.cpp

```diff
@@ -16,11 +16,11 @@ namespace meta {
 
 kvstore::ResultCode ActiveHostsMan::updateHostInfo(kvstore::KVStore* kv,
                                                    const HostAddr& hostAddr,
-                                                   const HostInfo& info) {
+                                                   const TimeInfo& info) {
     CHECK_NOTNULL(kv);
     std::vector<kvstore::KV> data;
     data.emplace_back(MetaServiceUtils::hostKey(hostAddr.first, hostAddr.second),
-                      HostInfo::encode(info));
+                      TimeInfo::encode(info));
```

Comment:
Why change the name?

Reply:
The ActiveHostsMan and LastUpdateTimeMan are all need to record the time, which is read(decode)/write(encode) the kvstore of metad.  So I extract the struct and change its name from `HostInfo` to `TimeInfo`.

Reply:
The `ActiveHostsMan` and `LastUpdateTimeMan` have been moved  to the file ActiveMan.h.